src/app.types.tsx

```diff
@@ -316,6 +316,13 @@ export interface UsersDict {
   users: User[];
 }
 
+export interface UserPatch {
+  _id: string; // Maps the `username` field in Python, which has an alias "_id"
+  updated_password?: string | null;
+  add_authorised_routes?: string[] | null;
+  remove_authorised_routes?: string[] | null;
```

Comment:
Might be worth at this point extracting an `AuthorisedRoutes` type that can be shared both here and in the `User`/`UserPost` interfaces.

Reply:
I've as used `extends Pick<UserPost, '_id'>` for the userPatch. To ensure that the `_id` is consistent across both types